charts/agent/values.yaml

```diff
@@ -21,6 +21,16 @@ cluster:
     enabled: true
   metrics:
     enabled: true
+    pod:
+      enabled: false
+      action: keep
+      interval: 10s
+      # add whatever namespace the collection is deployed to
+      namespace_drop_regex: (.*istio.*|.*ingress.*|kube-system|observe)
+      namespace_keep_regex: (.*)
```

Comment:
how does this interact with namespace_drop_regex?

Reply:
they act independently